lib/diagnostic.rb

```diff
@@ -105,15 +120,24 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+
+  def roar
+    puts 'ROAR!'
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# mixins are more of behaviors, whereas if something has direct inheritance,
```

Comment:
The primary reason is that classes can have many modules included, but only inherit from a single class.